src/Modera/BackendSecurityBundle/Controller/UsersController.php

```diff
@@ -73,12 +73,26 @@ public function getConfig()
                             'firstName' => $user->getFirstName(),
                             'lastName' => $user->getLastName(),
                             'middleName' => $user->getMiddleName(),
+                            'isActive' => $user->getIsActive(),
```

Comment:
Though method "setIsActive" and "getIsActive" were introduced in a scope of other commit it doesn't exactly follow proper mutator method names convention, instead it is preferred to have names such as "setActive" and "isActive" for boolean getter/setter pairs, please add @deprecated annotation to both setIsActive/getIsActive methods in User class and add proper ones (isActive, setActive) instead.